test/test_backend.py

```diff
@@ -170,7 +171,99 @@ def test_counts_over_time(self):
+        edges2 = subgraph([1738170,19122186])
+        self.assertEqual(edges2,
+                         [
+                             (
+                                 "1738170",
+                                 "1738203"
+                             ),
+                             (
+                                 "1738203",
+                                 "19122186"
+                             )
+                         ])
+        #Test for a more complex hierarchial relationship
+        edges3 = subgraph([321588,4027255,43530856,4024552,316139,45766164,43530961])
+        self.assertEqual(edges3,
+                         [
+                             (
+                                 "4024552",
+                                 "316139"
+                             ),
+                             (
+                                 "316139",
+                                 "43530961"
+                             ),
+                             (
+                                 "316139",
+                                 "45766164"
+                             ),
+                             (
+                                 "321588",
+                                 "4024552"
+                             ),
+                             (
+                                 "321588",
+                                 "4027255"
+                             ),
+                             (
+                                 "4027255",
+                                 "43530856"
+                             )
+                         ]
+                         )
+        #Testing a relationship where a common ancestor is needed to connect the graph
+        edges3 = subgraph([4027255, 43530856, 4024552, 316139, 45766164, 43530961])
+        self.assertEqual(edges3,
+                         [
+                             (
+                                 "4024552",
+                                 "316139"
+                             ),
+                             (
+                                 "316139",
+                                 "43530961"
+                             ),
+                             (
+                                 "316139",
+                                 "45766164"
+                             ),
+                             (
+                                 "321588",
+                                 "4024552"
+                             ),
+                             (
+                                 "321588",
+                                 "4027255"
+                             ),
+                             (
+                                 "4027255",
+                                 "43530856"
+                             )
+                         ]
+                         )
+            
 # Uncomment this and run this file directly to run all tests
-# if __name__ == '__main__':
-#     unittest.main()
+if __name__ == '__main__':
```

Comment:
Pretty sure this is on your radar to change this back to `unittest.main()` at the end, but just making note of it.